corehq/apps/app_manager/views/multimedia.py

```diff
@@ -27,3 +30,17 @@ def multimedia_ajax(request, domain, app_id):
         return render(request, "app_manager/partials/settings/multimedia_ajax.html", context)
     else:
         raise Http404()
+
+
+@require_deploy_apps
+@quickcache(['domain', 'app_id'], timeout=60 * 60)
```

Comment:
any thoughts on storing this on the app doc on the first calculation?
My understanding is that the multimedia size won't change for a build because the multimedia files are never deleted/updated. I don't want to pre-calculate and store it on the app doc because the use of this information is very limited.

Reply:
rather than storing it in the app doc what about having a separate doc so that the app doc doesn't get more bloated?

Reply:
I was thinking app doc to avoid another doc fetch/query, plus the information is a dict of 4 values which does not add much load to app doc but definitely would like to avoid adding more to it, even if its small.
If that's okay, should not this be an sql model instead, I thought we were moving away from using more of couch?

Reply:
Just how expensive is this to calculate? I'd rather calculate it dynamically than add one more piece of data to apps, regardless of whether it's on the app doc or a new model.

Reply:
It can take quite a while on ICDS size apps. I ran for ICDS AWW app for a build profile and it ran for few seconds. I can get a better estimate if you need.

Reply:
A couple seconds sounds reasonable to me, especially given that this is an ajax request.

Reply:
sorry, few seconds would be like 20+ seconds. I will follow up with a definite number

Reply:
Hrm, honestly even at 20 seconds I'm still inclined to calculate it dynamically and find another way to improve performance (not in this PR). This page already takes ages to load on ICDS.